src/main/pages/che-7/end-user-guide/proc_running-an-existing-workspace-from-the-user-dashboard-using-the-recent-workspaces.adoc

```diff
@@ -0,0 +1,20 @@
+[id="running-an-existing-workspace-from-the-user-dashboard-usinng-the-recent-workspaces_{context}"]
+= Running an existing workspace from the User Dashboard using the Recent Workspaces
+
+This section describes how to run an existing workspace from the User Dashboard using the Recent Workspaces.
+
+[discrete]
+== Prerequisites
+
+* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:../che-quick-starts[Che quick-starts].
```

Comment:
```suggestion
* A running instance of Eclipse Che. To install an instance of Eclipse Che see link:{{site.baseurl}}/che-7/che-quick-starts[Che quick-starts].
```

Reply:
@themr0c, please, use the `{{site.baseurl}}` var for links. This relative inking isn't portable.

Reply:
Sorry, no, just `{site-baseurl}` -- an AsciiDoc attribute.

Reply:
Ack